Cargo.toml

```diff
@@ -1,15 +1,18 @@
 [package]
-
-name = "rss"
+name = "rss-rs"
 version = "0.3.1"
-authors = ["Corey Farwell <coreyf@rwell.org>"]
-license = "Apache-2.0"
-repository = "https://github.com/frewsxcv/rust-rss"
-documentation = "https://frewsxcv.github.io/rust-rss/rss/index.html"
-description = "Library for serializing the RSS web content syndication format"
-keywords = ["rss", "feed", "blog", "web", "news"]
-exclude = ["test-data/"]
+authors = ["James Hurst <jh.jameshurst@gmail.com>"]
+description = "A fast RSS feed parser"
+repository = "https://github.com/jameshurst/rss-rs"
+documentation = "https://jameshurst.github.io/rss-rs/rss/"
+license = "MIT"
```

Comment:
No strong opinions on the license transition from Apache-2.0 to MIT (nowadays, I usually dual license Apache-2.0 *and* MIT)